src/test/java/org/broadinstitute/consent/http/db/DarCollectionSummaryDAOTest.java

```diff
@@ -723,6 +738,68 @@ void testGetDarCollectionSummaryTwoDataAccessRequests(String type) {
     assertTrue(summary.getDatasetIds().contains(dataset.getDatasetId()));
   }
 
+  @ParameterizedTest
+  @ValueSource(strings= {"DatasetIds", "CollectionId"})
+  void testGetDarCollectionSummaryTwoDataAccessRequestsForDACsWithVotes(String type) {
```

Comment:
I'm finding this test challenging to follow. I think it would be easier to read if some common setup functions were broken out into separate helper methods. For example, this is how I read what this test is doing:
1. create approved data access request with election, votes, and datasets
2. create approved PR from approved DAR (i.e. with new election, new votes, and DAR's dataset(s))
3. test `darCollectionSummaryDAO.getDarCollectionSummariesForDAC` in its own method
4. test `darCollectionSummaryDAO.getDarCollectionSummaryForDACByCollectionId` in its own method

Reply:
@raejohanek  - Let me know if you think it would be helpful for me to land my work first so you can leverage some of the new helpers that I've made there.  Otherwise, I'll hold off merging mine to deconflict the merge until after you're set.